src/main/java/net/mrscauthd/beyond_earth/block/RocketLaunchPad.java

```diff
@@ -31,6 +32,32 @@
 
     public static final BooleanProperty WATERLOGGED = BlockStateProperties.WATERLOGGED;
     public static final BooleanProperty STAGE = BlockStateProperties.LIT;
+    private static final List<Pair<Integer, Integer>> x = new ArrayList<>(List.of(
+            Pair.of(0, 1),
+            Pair.of(1, 1),
+            Pair.of(1, 0),
+            Pair.of(-1, 0),
+            Pair.of(-1, -1),
+            Pair.of(0, -1),
+            Pair.of(1, -1),
+            Pair.of(-1, 1)
+    ));
+    private static final List<Pair<Integer, Integer>> y = new ArrayList<>(List.of(
+            Pair.of(0,2),
+            Pair.of(0,-2),
+            Pair.of(2,0),
+            Pair.of(-2,0),
+            Pair.of(-2,1),
+            Pair.of(-2,-1),
+            Pair.of(2,1),
+            Pair.of(1,-2),
+            Pair.of(2,-2),
+            Pair.of(-2,-2),
+            Pair.of(-1,-1),
+            Pair.of(3,1),
+            Pair.of(1,2),
+            Pair.of(2,2)
+    ));
```

Comment:
By putting `x` and `y` here, it doesn't need to create a new list and add stuff to it every tick (a tick happens 20 times a second). Putting it here means it only makes the list once and then it is reused every tick rather than recreated.

Using `List.of(item, item, item...)` lets us add all the items to the List in one go rather than calling `y.add()` for every item.